sdk/eventhub/Azure.Messaging.EventHubs/src/Processor/EventProcessorClient.cs

```diff
@@ -11,7 +11,9 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Azure.Core;
+using Azure.Core.Pipeline;
 using Azure.Messaging.EventHubs.Core;
+using Azure.Messaging.EventHubs.Diagnostics;
 
 namespace Azure.Messaging.EventHubs.Processor
```

Comment:
nit:  This should move to the `Azure.Messaging.EventHubs` namesapce while we're at it.